src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -379,6 +379,12 @@ std::vector<std::string> disabledTestPatterns() {
     retVector.emplace_back(R"(.*smoke_RoPETest.*)");
 #endif
 
+#if defined(OPENVINO_ARCH_ARM64)
+    // to align with reference and c++ standard, Snippets PropagatePrecision should insert ConvertTruncation
+    // instead of ConvertSaturation when converting larger integer to smaller integer
+    retVector.emplace_back(R"(.*smoke_EltwiseChain_MergeConvert_int8/.*Op0=Prod.*Conversion=i8.*)");
+#endif
```

Comment:
As we discussed offline: need to add `Min` and `Max` ops to the model output to emulate saturation

Reply:
Thanks Alexandra for the comment! I think Dmitry's proposal to separate the FQ cases (use ConvertSaturation) with low-precision conversion cases from original framework (use ConvertTruncation) is great! And I'm having some more discussion with Dmitry!

I've tried add Min ops to produce saturated results for reference model. But the issue is truncation has already been done as a part of c++ multiply of int8. Extra Min will not change the reference result. Another possible way is to apply f32 precision for reference model and convert to int8 in the end. But as the test class EltwiseChainTest share models for target and reference, as it is not inherited from SnippetsFunctionBase. it will require much efforts to implement such Saturation behavior for reference model.

The more important thing is, as the flag with_quantization=false https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/arm/eltwise_chain.cpp#L37, the failed test case is actually the low-precision conversion case from original framework that requires truncation, which should be tracked by ticket 149216 (I created it to track the original framework case as Dmitry suggested). And when the separate ticket 149216 is resolved, this test case will pass without any modification of reference model.

So for now I skipped this test case and provided the ticket number to track it.
Please feel free to have further discussion if you have concern. Thanks @dmitry-gorokhov @a-sidorova !